src/GLM.jl

```diff
@@ -9,6 +9,9 @@ module GLM
     using StatsBase: StatsBase, CoefTable, StatisticalModel, RegressionModel
     using StatsFuns: logit, logistic
     using Distributions: sqrt2, sqrt2π
+    if VERSION >= v"0.6"
```

Comment:
Ah, I think Steven Johnson's recent SpecialFunctions change broke that. I can submit a PR there to fix it.